packages/core/src/app/shipping/SingleShippingForm.tsx

```diff
@@ -334,7 +344,7 @@ export default withLanguage(
             getFields,
             methodId,
         }: SingleShippingFormProps & WithLanguageProps) =>
-            methodId
```

Comment:
Could you please explain what's wrong with the previous control flow?
It looks to me that now we only apply `getCustomFormFieldsValidationSchema` to two IDs listed here, previously, we apply it to all IDs.

Reply:
AFAIK we are ignoring `PaymentMethodId.BraintreeAcceleratedCheckout and PaymentMethodId.PayPalCommerceAcceleratedCheckout` from custom validation, hence this method.

Is that the correct understanding @bc-nick ?

Reply:
We do not need **getCustomFormFieldsValidationSchema** for both methods because it filters by `custom` field and validation works incorrectly

You can take a look here: https://github.com/bigcommerce/checkout-js/blob/aa4bee8cece2b5efe775c731eb6481d4da480c6c/packages/core/src/app/formFields/getCustomFormFieldsValidationSchema.ts#L47

So we need to use default validation